### PR TITLE
Update IRN + KOR reg code and results w latest policies

### PR DESCRIPTION
**Pull request by jeanettelt**

### CODE REVIEW

File: codes/models/alt_growth_rates/IRN_adm1.do [hunk 1]
```diff
@@ -172,27 +173,34 @@ graph drop hist_irn qn_irn
 
 // predicted "actual" outcomes with real policies
 *predict y_actual if e(sample)
-predictnl y_actual = p_1*_b[p_1] + p_2* _b[p_2] + p_1_x_Tehran*_b[p_1_x_Tehran] + p_2_x_Tehran*_b[p_2_x_Tehran] ///
-+ _b[_cons] + __hdfe1__ + __hdfe2__ if e(sample), ci(lb_y_actual ub_y_actual)
+predictnl y_actual = ///
```
Comment (sannanphan) [inline codes/models/alt_growth_rates/IRN_adm1.do:176]: I think we need the testing regime here

Comment (sannanphan) [inline codes/models/alt_growth_rates/IRN_adm1.do:176]: We need to add testing regime in here. It is not in the treatment equation but should be in the predicted formula. I checked and it is missing in KOR too (my bad)

File: codes/models/alt_growth_rates/IRN_adm1.do [hunk 2]
```diff
@@ -172,27 +173,34 @@ graph drop hist_irn qn_irn
...
+p_1 * _b[p_1] + ///
+p_2 * _b[p_2] + ///
+p_1_x_Tehran * _b[p_1_x_Tehran] + ///
+p_2_x_Tehran * _b[p_2_x_Tehran] + ///
+p_3 * _b[p_3] ///
 if e(sample)
 
 // predicting counterfactual growth for each obs
 *gen y_counter = y_actual - treatment if e(sample)
 predictnl y_counter =  _b[_cons] + __hdfe1__ + __hdfe2__ if e(sample), ci(lb_counter ub_counter)
```
Comment (sannanphan) [inline codes/models/alt_growth_rates/IRN_adm1.do:196]: same thing here, should add testing regime

File: codes/models/alt_growth_rates/KOR_adm1.do [hunk 3]
```diff
@@ -202,6 +202,7 @@ p_1 * _b[p_1] + ///
 p_2 * _b[p_2] + ///
 p_3 * _b[p_3] + ///  
 p_4 * _b[p_4] + ///  
+p_5 * _b[p_5] + ///
```
Comment (sannanphan) [inline codes/models/alt_growth_rates/KOR_adm1.do:205]: same as IRN, we need to add testing regime in y_actual and y_counter, but not in treatment

File: codes/models/alt_growth_rates/IRN_adm1.do (hunk 1 above)
Comment (jeanettelt) [reply codes/models/alt_growth_rates/IRN_adm1.do]: fixed in all countries

File: codes/models/alt_growth_rates/IRN_adm1.do (hunk 2 above)
Comment (jeanettelt) [reply codes/models/alt_growth_rates/IRN_adm1.do]: fixed in all countries

File: codes/models/alt_growth_rates/KOR_adm1.do (hunk 3 above)
Comment (jeanettelt) [reply codes/models/alt_growth_rates/KOR_adm1.do]: fixed in all countries

**Pull request merged**